source/_integrations/climate.mqtt.markdown

```diff
@@ -456,3 +456,45 @@ mqtt:
       swing_mode_command_topic: "study/ac/swing/set"
       precision: 1.0
 ```
+
+## Example of multiple devices
+
+{% raw %}
+
+```yaml
+mqtt:
+  climate:
+    - name: Study
+      modes:
+        - "off"
+        - "heat"
+        - "auto"
+      mode_command_topic: "study/ac/mode/set"
+      mode_state_topic: "study/ac/mode/state"
+      mode_state_template: "{{ value_json }}"
+
+    - name: Living
+      modes:
+        - "off"
+        - "auto"
+        - "cool"
+        - "heat"
+        - "dry"
+        - "fan_only
+      swing_modes:
+        - "on"
+        - "off"
+      fan_modes:
+        - "quiet"
+        - "lvl_1"
+        - "lvl_2"
+        - "lvl_3"
+        - "lvl_4"
+        - "lvl_5"
+        - "auto"
+      mode_command_topic: "living/ac/mode/set"
+      mode_state_topic: "living/ac/mode/state"
+      mode_state_template: "{{ value_json }}"
+```
```

Comment:
This example is extremely large for what it is trying to display/show/learn.

Additionally, it is documented and shown in multiple places that `climate` is a list. A YAML list can just be added multiple items too. This is not specific to Home Assistant, but how lists in YAML work.

I'm going to friendly decline this PR for those two reasons.

../Frenck